perfkitbenchmarker/providers/alicloud/util.py

```diff
@@ -40,6 +40,17 @@
 REGION_HZ = 'cn-hangzhou'
 
 
+ADD_USER_TEMPLATE = '''#!/bin/bash
+echo "{0} ALL = NOPASSWD: ALL" >> /etc/sudoers
+useradd {0} --home /home/{0} --shell /bin/bash -m
+mkdir /home/{0}/.ssh
+echo "{1}" >> /home/{0}/.ssh/authorized_keys
```

Comment:
Can you use {user_name} and {public_key} for readability?

Reply:
Done.